docs/release-notes/cloud/cloud-2024-05.md

```diff
@@ -0,0 +1,94 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Cloud May 2024.
+included_jira_versions: ['2024.05']
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/7.10.305
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/7.14.11
+---
+
+# Cloud May 2024
+
+These release notes are for the Codacy Cloud updates during May 2024.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Product enhancements
+
+-   You can now [quickly create a new coding standard](../../organizations/using-coding-standards.md#creating) by activating code patterns in bulk from preset category groups. This option is also available when setting up a new organization to create a default coding standard. (ALA-824)
```

Comment:
@nicklem Small tweak to solve Vale check and simplify some technical jargon.
Let me know if it makes sense. 🙏 

```suggestion
-   You can now [create a new coding standard](../../organizations/using-coding-standards.md#creating) by adjusting the sensitivity of preset category groups. This option is also available when setting up a new organization to create a default coding standard. (ALA-824)
```

Reply:
Looks great! Thanks! 🫶